### PR TITLE
Fix docs for update security index settings API 

### PR DESCRIPTION
**Pull request by n1v0lg**
Docs tweak with a typo fix and a clarification on how the two available settings interact (essentially https://github.com/elastic/elasticsearch/issues/27871). I'm also open to including this info in the more generic settings API but feels like a simple enough callout to add to the security API.

### CODE REVIEW

**Comment by n1v0lg** [comment]
@elasticmachine update branch

File: docs/reference/rest-api/security/update-settings.asciidoc [hunk 1]
```diff
@@ -18,27 +18,28 @@ Updates the settings of the security internal indices.
...
 configuration, including Native realm users and roles configured via the API.
 
 `security-tokens`::
-(Optional, object) Settings to be used for the index used to store 
+(Optional, object) Settings to be used for the index used to store
 <<security-api-get-token,tokens>>.
 
-`security`::
-(Optional, object) Settings to be used for the index used to store 
+`security-profile`::
```
Comment (n1v0lg) [inline docs/reference/rest-api/security/update-settings.asciidoc:28]: This is change 1.

File: docs/reference/rest-api/security/update-settings.asciidoc [hunk 2]
```diff
@@ -18,27 +18,28 @@ Updates the settings of the security internal indices.
...
-This API allows a user to modify the settings for the Security internal indices 
-(`.security` and associated indices). Only a subset of settings are allowed to 
+This API allows a user to modify the settings for the Security internal indices
+(`.security` and associated indices). Only a subset of settings are allowed to
 be modified. This includes:
 
 - `index.auto_expand_replicas`
 - `index.number_of_replicas`
 
+NOTE: If `index.auto_expand_replicas` is set, `index.number_of_replicas` will be ignored during updates.
```
Comment (n1v0lg) [inline docs/reference/rest-api/security/update-settings.asciidoc:42]: This is change 2: I _think_ it's as simple as that. Basically, if you have e.g., 0-1 for auto expand, and a 1 node cluster, trying to set number of replicates to say 2 won't have any effect (https://github.com/elastic/elasticsearch/issues/27871)

File: docs/reference/rest-api/security/update-settings.asciidoc (hunk 1 above)
Comment (gwbrown) [reply docs/reference/rest-api/security/update-settings.asciidoc:28]: Truly, the most pernicious class of error: the copy/paste error.

🤦🏼‍♀️ thanks

**Review by AthenaEryma — APPROVED** [review]
LGTM, thanks!

File: docs/reference/rest-api/security/update-settings.asciidoc (hunk 2 above)
Comment (gwbrown) [reply docs/reference/rest-api/security/update-settings.asciidoc:42]: Matches my understanding, and this API just plumbs through to the normal settings infra so it should behave the same as the same setting on any other index.

**Comment by n1v0lg** [comment]
@elasticmachine update branch

**Pull request merged**